core/dbt/cli/main.py

```diff
@@ -26,12 +42,21 @@
 from dbt.task.test import TestTask
 
 
-class dbtUsageException(Exception):
-    pass
+@dataclass
+class DbtRunnerResult:
```

Comment:
Nit: It feels weird that this dataclass is named `DbtRunnerResult` when the class is named `dbtRunner`.

I know the latter goes against TitleCase for class names, but as a company/brand, we feel very very strongly about lowercasing `dbt`. I know we're not perfectly consistent about this in the codebase today :)